src/index.html

```diff
@@ -5,9 +5,61 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="Imac picture"
+        class="card__image"
+        >
```

Comment:
```suggestion
      <img
        src="images/imac.jpeg"
        alt="APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)"
        class="card__image"
      >
```
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md):
13. [SEO] - alt atribute should describe the image if the image contains information (better description you have - better for you :))
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/6aee76be-bff0-4059-a9c4-91bebcdf0a69)
14. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.